routes/dao_coin_exchange.go

```diff
@@ -789,6 +839,16 @@ func formatFloatAsString(f float64) string {
 	return fmt.Sprintf("%d.0", fAsBigInt)
 }
 
+// Returns float64 representation of the input decimal string. If the input is not a valid decimal string, this will
+// always return 0
+func tryParseFloatFromDecimalString(s string) (float64, error) {
+	float, err := strconv.ParseFloat(s, 64)
+	if err != nil {
+		return 0, err
+	}
+	return float, nil
+}
```

Comment:
This function is a little unnecessary - ParseFloat will return 0 if there's an error as the first return value.